Postresql v 9.4.1211 OSGi Activation is excessively noisy

After starting openHAB with jdbc persistance and postgresql version 9.4.1211, the SQL Driver logs irrelevant, possibly misleading messages to the console:

```
~/projects/openhab/openhab$ ./start.sh 
Launching the openHAB runtime...
java.lang.IllegalArgumentException: Unable to load DataSourceFactory. Will ignore DataSourceFactory registration. If you need one, ensure org.osgi.enterprise is on the classpath
	at org.postgresql.osgi.PGBundleActivator.start(PGBundleActivator.java:38)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:771)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:1)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:764)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:721)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:936)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:319)
	at org.eclipse.osgi.container.Module.doStart(Module.java:571)
	at org.eclipse.osgi.container.Module.start(Module.java:439)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1582)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1562)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1533)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1476)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340)
Caused by: java.lang.NoClassDefFoundError: org/osgi/service/jdbc/DataSourceFactory
	at org.postgresql.osgi.PGBundleActivator.start(PGBundleActivator.java:33)
	... 16 more
Caused by: java.lang.ClassNotFoundException: org.osgi.service.jdbc.DataSourceFactory cannot be found by org.postgresql.jdbc42_9.4.1211
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:432)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:345)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:337)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:160)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 17 more

                          __  _____    ____      
  ____  ____  ___  ____  / / / /   |  / __ )     
 / __ \/ __ \/ _ \/ __ \/ /_/ / /| | / __  | 
/ /_/ / /_/ /  __/ / / / __  / ___ |/ /_/ /      
\____/ .___/\___/_/ /_/_/ /_/_/  |_/_____/     
    /_/                        2.0.0-SNAPSHOT
                               Build #516   

Hit '<tab>' for a list of available commands
and '[cmd] --help' for help on a specific command.
Hit '<ctrl-d>' or type 'system:shutdown' or 'logout' to shutdown openHAB.

openhab> 
```
This is not an error, but just a message; rather confusing...

## Possible Solution
Bump postgresql version to 9.4.1212. Herein, the above messages are suppressed, unless debugging the driver with -Dpgjdbc.osgi.debug=true (https://github.com/pgjdbc/pgjdbc/issues/672).

## Steps to Reproduce (for bugs)
1. Drop postgresql in the  9.4.1211 into addons  directory
2. Fire up openhab

--- a/pom.xml
+++ b/pom.xml
@@ -63,7 +63,7 @@
 		<hsqldb.version>2.3.3</hsqldb.version>
 		<mariadb.version>1.3.5</mariadb.version>
 		<mysql.version>5.1.38</mysql.version>
-		<postgresql.version>9.4.1211</postgresql.version>
+		<postgresql.version>9.4.1212</postgresql.version>
 		<sqlite.version>3.8.11.2</sqlite.version>
 
     </properties>

